towncrier/newsfragments/1962.bugfix

```diff
@@ -0,0 +1,4 @@
+Reverse proxy documentation has been updated to reflect new security hardening from PR#1959.
+If you do not set the required header, Mailu will not have access to the real ip address of the connecting client.
```

Comment:
I think it's misleading: more than the required header (That may have already been configured), it's the configuration parameters in Mailu that need to be set